src/wasm/literal.cpp

```diff
@@ -75,6 +75,7 @@ int64_t Literal::getBits() const {
   switch (type) {
     case Type::i32: case Type::f32: return i32;
     case Type::i64: case Type::f64: return i64;
+    case Type::v128: assert(false && "v128 not implemented");
```

Comment:
I think we should consider what to do with this function. Perhaps rename it to `getBits64`, and then a v128 would be an error here.